package.json

```diff
@@ -21,8 +21,9 @@
     "clean-full:package_modules": "rimraf ./packages/*/node_modules",
     "generate:typings": "node ./scripts/type-generation/generate-all-typings.js",
     "link": "node ./scripts/link",
-    "release": "NODE_ENV=production npm run setup-full && npm run generate:typings && lerna publish --exact",
+    "release": "NODE_ENV=production npm run setup-full && npm run generate:typings && lerna publish --exact && npm run pushTags",
     "release:next": "npm run release -- --dist-tag next",
+    "pushTags": "./scripts/pushTags.js",
```

Comment:
Could you configure this as the postPublish script?

Reply:
good idea 👍